pw_mgr/lib/crypto_utils.py

```diff
@@ -0,0 +1,63 @@
+from cryptography.fernet import Fernet
+
+
+def write_key():
```

Comment:
you might want to pass in a variable that is the location so you can have different locations for tests and locations for running stuff

Reply:
Right, i saw the comment and spent the morning reading up on using __file__. I think I get the concept: define a path so the program knows where to look for and/or create the .txt and .key files. Now I just need to figure out the code to make it happen